app/assets/javascripts/social_networking/controllers/goal-controller.js

```diff
@@ -11,11 +11,14 @@
     this.studyEndDate = studyEndDate;
     this.noticesEnabled = noticesEnabled;
     this.noticeUtility = noticeUtility;
+    this.displayPopoverHTML = displayPopoverHTML || false;
 
     this.resetForm();
     this.resetTabs();
     if (typeof $ !== 'undefined') {
-      $("#help-pop").popover();
+      $("#help-pop").popover({
+        html: this.displayPopoverHTML
```

Comment:
@ericcf quick question.  We need to have html in this tool tip.  It seems that it would be easier to just make this help tool tip have a default of `html: true` instead of adding another dependency.  Thoughts?   :brain Thanks!